tests/test_settings.py

```diff
@@ -11,5 +11,5 @@
 DEV_MODE = True
 
 # server to check against
-# SITE_HOST = 'localhost:8080'
-HOSTNAME = 'localhost/nd'
+HOSTNAME = 'localhost:8080'
```

Comment:
This should be localhost/nd when being pushed, as the travis server (and normal production servers) do not run on port 8080